service-tests/vscode.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'vscode', title: 'VS Code Marketplace' });
+module.exports = t;
+
+t.create('installs should be formatted. eg. 72M')
+.get('/installs/ritwickdey.LiveServer.json')
```

Comment:
Could you change `installs` to `d`, `version` to `v`, and `rating` to `r` for consistency? Sorry I missed this before!

Reply:
Okay. no problem.. I'll :smile: